.ipynb_checkpoints/nflscrapeR Notebook-checkpoint.ipynb

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Add `.ipynb` checkpoints to gitignore:
https://stackoverflow.com/questions/35916658/how-to-git-ignore-ipython-notebook-checkpoints-anywhere-in-repository

When you create a new repo in github select `Python` for `gitignore` and all of these files are automatically ignored.